api/integrations/launch_darkly/launch_darkly.py

```diff
@@ -0,0 +1,239 @@
+from datetime import datetime
+from typing import Optional
+from uuid import UUID
+
+from django.db import IntegrityError
+
+from environments.models import Environment
+from features.feature_types import STANDARD
+from features.models import Feature, FeatureState, FeatureStateValue
+from features.multivariate.models import (
+    MultivariateFeatureOption,
+    MultivariateFeatureStateValue,
+)
+from features.value_types import BOOLEAN, FLOAT, INTEGER, STRING
+from integrations.launch_darkly.client import LaunchDarklyClient
+from integrations.launch_darkly.models import LaunchDarklyImport
+from integrations.launch_darkly.serializers import LaunchDarklyImportSerializer
+from organisations.models import Organisation
+from projects.models import Project
+from users.models import FFAdminUser
+
+
+class LaunchDarklyWrapper:
+    def __init__(
+        self,
+        api_key: str,
+        user: FFAdminUser,
+        request: LaunchDarklyImportSerializer,
+        client: LaunchDarklyClient = None,
+    ):
+        self.client = client or LaunchDarklyClient(api_key)
+        self.request = request
+        self.logger = LaunchDarklyImport.objects.create(
+            user=user,
+            organisation=self.request["organisation_id"],
+            project=self.request["project_id"],
```

Comment:
Looks like `project_id` isn't required so there might be the possibility of a keyerror here?